.github/workflows/dockerfile-lint.yml

```
@@ -0,0 +1,14 @@
+name: Dockerfile Lint
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: lint
+      uses: luke142367/Docker-Lint-Action@v1.0.0
```

This isn't something we've gotten consensus on or anything, but I think we'll want to start moving to pinning commit SHAs for better security, like so:

```suggestion
      uses: luke142367/Docker-Lint-Action@5c4c86226f39785a66827bbc2e322600c9afa3a9 # v1.1.1
```

This way, if the action owner's account is ever compromised, we're not affected. (Apparently even using tags is not sufficient; tags can be deleted and recreated, and some Actions maintainers even do this on purpose as a sort of semver-ish thing!)

The other thing I noticed is that the Action's Dockerfile has `RUN npm install -g dockerfilelint` which is itself unpinned, and that package [hasn't been updated](https://www.npmjs.com/package/dockerfilelint) in two years; it may be unmaintained and is risky. Having the Action pin it would be better. But also, we may not want to use something that uses this outdated package.

(I also happened to notice some [unsafe string interpolation](https://github.com/luke142367/Docker-Lint-Action/blob/5c4c86226f39785a66827bbc2e322600c9afa3a9/build/main.js#L27). Nothing critical, but a flag that we might want to be cautious.)